src/linux_mcp_server/audit.py

```diff
@@ -31,6 +35,22 @@
 }
 
 
+class Event(StrEnum):
+    LOCAL_EXEC_ERROR = "LOCAL_EXEC_ERROR"
+    REMOTE_EXEC = "REMOTE_EXEC"
+    REMOTE_EXEC_ERROR = "REMOTE_EXEC_ERROR"
+    SSH_AUTH_FAILED = "SSH_AUTH_FAILED"
+    SSH_CONNECT = "SSH_CONNECT"
+    SSH_CONNECTING = "SSH_CONNECTING"
+    TOOL_CALL = "TOOL_CALL"
+    TOOL_COMPLETE = "TOOL_CALL"
+
+
+class ExecutionMode(StrEnum):
+    remote = "remote"
```

Comment:
Call me picky, but if we defined Event enum strings all with uppercase, why do't we do the same for local/remote? Just for consistency. Feel free to discard this NIT comment.

Reply:
I don't feel strongly one way on another. Enums in Python are more akin to attributes than constants so the capitalization is a lot more flexible. By convention, module globals are usually the only thing that is uppercased (there are common exceptions to that, though).

I mainly prefer snake case for attributes and that's how I think of enum values in Python. I only uppercased the attributes on `Event` to match the values. I would honestly rather those be snake case as well.